.ctk/README.md

```
@@ -0,0 +1,180 @@
+import pytest
+from testcontainers.postgres import PostgresContainer
+
+@pytest.fixture(scope="session")
+def pg_url():
+    with PostgresContainer("postgres:16") as pg:
+        yield pg.get_connection_url()
+```
+
+Typical workflow: `./run_tests.sh unit` constantly while coding, full `./run_tests.sh` (or `-m "not slow"`) in CI.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)